BrightID/src/components/GroupsScreens/Members/MembersScreen.js

```diff
@@ -65,6 +66,36 @@ export class MembersScreen extends Component<Props, State> {
     );
   };
 
+  confirmAddAdmin = (user) => {
+    const buttons = [
+      {
+        text: 'Cancel',
+        style: 'cancel',
+      },
+      {
+        text: 'OK',
+        onPress: async () => {
+          const { route, dispatch } = this.props;
+          const group = route.params?.group;
+          try {
+            await api.addAdmin(user.id, group.id);
+            await dispatch(addAdmin(user.id, group));
+          } catch (err) {
+            Alert.alert('Error dismissing member from the group', err.message);
```

Comment:
This alert message doesn't seem to match the context of the error

Reply:
Nice find!